doc/Makefile

```diff
@@ -43,7 +43,8 @@ man:
 	done
 	$(OPAM_INSTALLER) $(HELPFMT) > man/opam-installer.1 2>> man/err
 # Dune doesn't (yet) support --no-print-directory
```

Comment:
Is there an upstream dune pr for this `--no-print-directory` feature? Worth noting in the comment if so.

Reply:
It's in Dune 1.7.0 - on the `master` branch PR (coming shortly) for Dune 1.10, the comment is gone...